### ISSUE TITLE
Proof: use GenAI to extract price tag data

### ISSUE THREAD

**Issue by raphodn** [openfoodfacts/open-prices#569]
### Story

Following a discussion in #560
And in parallel of #320 & #511

Use GenAI APIs to extract data. 

### GenAI providers

- Google Gemini
- ?

**Comment by raphodn** [comment]
The follow-up issue was https://github.com/openfoodfacts/open-prices/issues/611 to start storing this info in a dedicated `PriceTagPrediction` table, and display these predictions to users for price validation
